docs/source/en/optimization/fp16.mdx

```diff
@@ -251,6 +253,10 @@ image = pipe(prompt).images[0]
 This feature requires `accelerate` version 0.17.0 or larger.
 </Tip>
 
+**Note**: `enable_model_cpu_offload()` is a stateful operation that installs hooks on the models and state on the pipeline. In order to properly re-offload
+models after they are called, it is required that the entire pipeline is run and models are called in the order the pipeline expects them to be. Exercise caution
+if models are re-used outside the context of the pipeline after hooks have been installed.
```

Comment:
Could we also include notes on discarding the installed hooks?

Reply:
https://github.com/huggingface/diffusers/pull/3252/commits/c099a8fa5608231c991c5608ac4bf5df6b194b39